module/VuFind/src/VuFind/Role/PermissionProvider/PluginManager.php

```diff
@@ -50,7 +50,8 @@ class PluginManager extends \VuFind\ServiceManager\AbstractPluginManager
         'ipRange' => IpRange::class,
         'ipRegEx' => IpRegEx::class,
         'role' => Role::class,
-        'serverParam' => ServerParam::class,
+    'serverParam' => ServerParam::class,
```

Comment:
There still seems to be some kind of whitespace setting problem somewhere that's causing indentation to get lost. Can you check your editor settings?

Reply:
thanks @demiankatz I think I've found the offending setting in my editor and have been able to see and fix this up.